### ISSUE TITLE
[webUI] User cannot share space with group

### ISSUE THREAD

**Issue by grgprarup** [owncloud/web#8112]
### Description
Users can not share space with the group.
More description can be found here https://github.com/owncloud/ocis/issues/5036#issuecomment-1336362711

### Steps to reproduce
1. Create user `Alice`, `Brian`, `Carol`
2. As admin, create group `group1`
3. As admin, add users `Brian` and `Carol` to group `group1`
4. As admin, give `Space Admin` role to user `Alice`
5. As Alice create space `space-to-share`
6. As Alice, share space `space-to-share` to group `group1`

### Expected behaviour
The backend allows users to share space with a group
Test added for server behavior in this PR https://github.com/owncloud/ocis/pull/5023

### Actual behaviour
Group is not found when sharing with group
![Screenshot from 2022-12-14 17-08-49](https://user-images.githubusercontent.com/8559121/207583131-d7b023ff-e5b1-413a-bbd2-12447f56f376.png)

### Environment general
**Backend (ownCloud Core or Infinite Scale)**: ocis (latest)

**Comment by JammingBen** [comment]
We already have an open issue for this: https://github.com/owncloud/web/issues/8035.

tl;dr: This is a feature request which will be shipped in the future (probably/hopefully soon).

I'm closing here in favour of #8035

**Comment by kulmann** [comment]
> We already have an open issue for this: #8035.
> 
> tl;dr: This is a feature request which will be shipped in the future (probably/hopefully soon).
> 
> I'm closing here in favour of #8035

We'll actually work on this feature request in one of the next sprints (maybe already the next one). 🚀
